### ISSUE TITLE
Moobloom-notes finalized

### ISSUE THREAD

**Issue by BrassMcCrafty** [BrassAmber-Mods/ClassicHostiles#33]
-added purely out of the many failed attempts and abandoned mods
-my partner also loves them

- [x] does not spawn as wither rose variants due to nether and wither being the opposite of bloom literally
- [x] spawns in meadows and flower forests
- [x] bees will pollinate off their backs
- [x] using a shear removes the flowers, a hoe will remove the color making it brown
- [x] mooblooms can be "tilled" with a hoe which turns them into tilled_mooblooms skin
- [x] they give Floral Fluid
- [x] fluid is used on a tilled moobloom to then use seeds on it, which will sprout a random flower on it

Biomes:
flower_forest
meadows
-no other biomes

So basically the floral fluid is an overload of toxins from the flowers and it makes your body go into shutdown mode
Floral Fluid:
- [ ] Sounds
- [ ] Custom texture?
- [ ] Item physics (Slower more lava like)
- [ ] Gives potion effects to entity in the fluid
- [x] Slow moving
- [ ] Bottles can be used for crafting Suspicious Stew
- [ ] Cauldron
- [ ] Should interact with lava and water?
- [ ] Turn dirt to grass when flowing overtop/touching block
- [ ] When dripping from dripstone -> Chance to create flowers
- [ ] Turn azalea leaves to flowering leaves

**Comment by Xratedjunior** [comment]
Ideas:

Just for now until we polish it up
shears- remove flower
hoe-remove floral coating
then you plant a flower on it
orr better yet
give the fluid some use
bottled fluid
right-click on a tilled cow to cause the zombie shakes
and then a random flower sprouts on it and it changes into a moobloom again

I also feel like bottles should maybe be usable in the crafting of suspicious stew?
A bucket seems much for only a bowl of stew
we can increase the 'milk' time on mooblooms

**Comment by Xratedjunior** [comment]
We could drop seeds or something random to emphasise the plant part

after using the hoe
its fluid [weakness potion for zombie]
and then seed [golden apple]
same process exact
different items
Xrated_junior — Today at 12:28
Any seed for crops? or just wheat seeds?

**Comment by Xratedjunior** [comment]
Flowers around Moobloom increase flower growth.
Maybe get increased result chance from flowers around the Moobloom?
